frontend/src/components/limited-textarea/index.tsx

```diff
@@ -0,0 +1,26 @@
+import { Text, Textarea, TextareaProps } from "theme-ui";
+
+type LimitedTextareaPros = TextareaProps & {
+  value: string;
+  limit: number;
+};
+
+export const LimitedTextarea = ({
+  value,
+  limit,
+  ...props
+}: LimitedTextareaPros) => {
+  return (
+    <>
```

Comment:
I wonder if for accessibility reason the wrapper should be a label 🤔

Reply:
I forgot about accessibility :( 
We should definitely have a look at it!

Reply:
I can put something like this in the Text
` aria-label={`${(value as string)?.length} characters left over ${maxLength}`}` 
but we need much more than that... we don't use any `aria-label` (don't know if there are other fields to improve  accessibility...) 
We can work on that in another issue I think